pkg/printer/enclave.go

```diff
@@ -214,7 +214,11 @@ func Secrets(secrets map[string]models.ComputedSecret, secretsToPrint []string,
 		vals := []string{}
 		for _, name := range secretsToPrint {
 			if secrets[name] != (models.ComputedSecret{}) {
-				vals = append(vals, secrets[name].ComputedValue)
+				if secrets[name].ComputedValue != nil {
+					vals = append(vals, *secrets[name].ComputedValue)
+				} else {
+					vals = append(vals, "")
```

Comment:
Setting the clipboard to an empty string is reasonable but returning an error seems like the more correct thing to do here.

Also we should consider moving the `copy` mechanism to the `getSecrets` func since that's the only thing using it.